src/provisioner/cluster.py

```diff
@@ -112,7 +112,6 @@ def _get_storage_kv_list(self, storage_spec: dict, node_id: str):
         # Create confstore keys,from storage_spec(config from config.yaml).
         storage_kv_list = []
         key_prefix = f'node>{node_id}>storage'
-        storage_kv_list.append((f'{key_prefix}>cvg_count', len(storage_spec)))
```

Comment:
Check with Motr team if they are using this already. If so, we need to ask them to switch. 
So, Please keep this for backward compatibility.

Reply:
added them back with todo comment for backward compatibility